docs/output.md

```diff
@@ -262,7 +262,7 @@ Protein-coding genes are predicted for each assembly.
 <details markdown="1">
 <summary>Output files</summary>
 
-- `Prodigal/`
+- `Annotation/Prodigal/`
   - `[sample/group].gff.gz`: Gene Coordinates in GFF format
   - `[sample/group].faa.gz`: The protein translation file consists of all the proteins from all the sequences in multiple FASTA format.
   - `[sample/group].fna.gz`: Nucleotide sequences of the predicted proteins using the DNA alphabet, not mRNA (so you will see 'T' in the output and not 'U').
```

Comment:
Shouldn't these also have assembler in the name :thinking:  (sorry not your original content, but still :sweat_smile:  )